root/user/UserProfile.js

```diff
@@ -71,44 +71,56 @@ function generateUserTypesList(
   }
   if (isAutoEditor(user)) {
     typesList.push(
-      <a href="/doc/Editor#Auto-editors" key="auto-editor">
+      <a href="/doc/Editor#Auto-editor" key="auto-editor">
         {lp('Auto-editor', 'user type')}
       </a>,
     );
   }
   if (isBot(user)) {
-    typesList.push(lp('Internal/Bot', 'user type'));
+    typesList.push(
+      <a href="/doc/Editor#Bot" key="bot">
+        {lp('Internal/Bot', 'user type')}
+      </a>,
+    );
+  }
+  if (isAccountAdmin(user)) {
+    typesList.push(
+      <a href="/doc/Editor#Account_admin" key="account-admin">
+        {lp('Account admin', 'user type')}
```

Comment:
Just curious, are these listed in any particular order? They don't seem to be sorted by power nor alphabetically. :)

Reply:
I have no idea why the existing order was what it is - my reasoning on the sorting of account admin was "auto-editor is the most relevant for a day-to-day user checking that, and after that account admin seems the most important due to the power it involves so I'll put it second" :man_shrugging: